scripts/start.js

```diff
@@ -86,41 +86,16 @@ choosePort(HOST, DEFAULT_PORT)
       urls.lanUrlForConfig
     );
 
-    //Default accounts to seed so we can have accounts with 1M ether for testing
-    var accounts =
```

Comment:
Can we at least extract this into a `start_ganache.js` script so that people can continue to use ganache in the same way, via `yarn start:ganache` or something like that?